src/mono/mono/mini/aot-compiler.c

```diff
@@ -306,7 +308,8 @@ typedef struct MonoAotCompile {
 	GHashTable *image_hash;
 	GHashTable *method_to_cfg;
 	GHashTable *token_info_hash;
-	GHashTable *method_to_pinvoke_import;
```

Comment:
No real need to change the name of this one? It is still pinvoke import data, just that we added more information to it and all usage have been updated, so think the old name could still be around.

Reply:
I was thinking the `import` specifically meant the `entrypoint` name, and wanted to clarify that it was going to hold both the scope and import, but I can put a comment to explicitly mention that the value in this hashtable corresponds to a string array of exactly a `scope` and `import`.